data/redis.py

```diff
@@ -0,0 +1,7 @@
+import json
+
+def cache_for_one_day(path_string, response, redis_client):
```

Comment:
since redis client is gloabl anyways, you can move it to this file. a later refactor we can save it as a class member or use the partial function